app/views/layouts/nsm/_primary_navigation.html.erb

```diff
@@ -0,0 +1,25 @@
+<% current ||= :your %>
```

Comment:
I think we can now delete app/views/layouts/_primary_navigation as you're replacing it with this new, better, appropriately namespaced one